tests/conftest.py

```diff
@@ -0,0 +1,3 @@
+def pytest_addoption(parser):
+    parser.addoption("--skip-slow-spec", action="store_true", required=False)
+    parser.addoption("--stop-after-command-line", type=int, required=False)
```

Comment:
So this one is primarily meant to be used when you're running a single test, and you are only interested in the first part of that test running? Seems pretty specific, but I guess I'm missing a common case. A bit more info about how/when to use it would help out people who haven't been living/breathing the test suite. :)

Reply:
I'll add some developer docs about running the spec tests before merging with examples of how to use these commands.

Reply:
As for *specificity*, a common case is that a spec test is failing in the `i32.wast` tests on command line 123 in a branch you are working on (but passing in master).  You run `pytest tests/spec/ -k "132.wast" --stop-after-command-line=123 --log-cli-level=debug` in `master`, save the log output, do the same in your branch, and then compare the results.

Without this, the log statements for command line 123 will be *very* deep in log history as many of these files have 1,000's of commands/lines.